src/components/CustomerQuota/CheckoutSuccess/RedeemedItem.tsx

```diff
@@ -18,7 +18,7 @@ export const RedeemedItem: FunctionComponent<{
       <View style={sharedStyles.itemRow}>
         <AppText style={sharedStyles.itemHeaderText}>{categoryName}</AppText>
       </View>
-      {identifierInputs && identifierInputs.length > 0 && (
+      {identifierInputs && identifierInputs.length > 1 && (
```

Comment:
what if there's only 1 input though? :x

Reply:
According to the TT screens, no need to display, so I just use this hahahaha.

Hmmmm but for TT are we doing 2 identifiers per item ah? If yes then this will be a problem~

Reply:
Maybe I can do this check

```
show identifier values if:
  identifierInputs.length > 0 && identifierInputs[0].type === identifierInputs[1].type
```

?

Reply:
Will change this to display the single identifier when there is only one, and a range when more than 2 for now?

Reply:
okk! sounds good~ ((:

Reply:
Updated :D